bot/qwen_code.py

```
@@ -0,0 +1,376 @@
+            # Check if comfytv MCP server is already configured
+            result = subprocess.run(
+                [*argv, "mcp", "list"],
+                capture_output=True,
+                text=True,
+                timeout=10,
+            )
+            if "comfytv" in result.stdout:
+                return  # Already configured
+
+            # Add the MCP server
+            subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)